src/option.c

```diff
@@ -1837,7 +1837,7 @@ stropt_get_newval(
 					     &(options[opt_idx]), OPT_GLOBAL));
     else
     {
-	++arg;	// jump to after the '=' or ':'
+	++arg;	// joption_value2stringump to after the '=' or ':'
```

Comment:
Looks like an accidental mistake

Reply:
Yeah it was pointed out in https://github.com/vim/vim/commit/900894b09a95398dfc75599e9f0aa2ea25723384#r128808467 as well. I have a follow-up PR that will fix this.